locales/en/index.yml

```diff
@@ -1582,11 +1583,24 @@ send_email_messages:
       info: You will not receive email messages anymore
     enable_all:
       label: Enable for all active services
-      info: Every message will be forwarded to your email, if allowed by the service
+      info: Every message will be forwarded to your epreferencesmail, if allowed by the service
```

Comment:
@CrisTofani is this a typo or a new variable?

Reply:
@thisisjp a typo, thanks to notice! 😄